uitest/README.md

```diff
@@ -0,0 +1,44 @@
+## Dev Home UI Testing
+### How to run UI tests?
+1. From the command prompt, start the Windows Application Driver service:
+```cmd
+"C:\Program Files\Windows Application Driver\WinAppDriver.exe"
+```
+2. From Visual Studio, navigate to the "Test Explorer" tab and locate the `DevHome.UITest` set of tests.
+3. Select a test and run it.
+    - Note: Once the test starts, avoid interacting with your machine (e.g. move mouse, use keyboard) to allow the test to navigate the Dev Home app and execute the test.
+
+### How to configure the application settings for a Dev Home release?
+Application settings are a set of configuration properties loaded from `appsettings.*.json` and are available at runtime to the test methods.
+For a developer release of Dev Home, the default configuration from `appsettings.json` are used. For Canary and Prod, a
+separate application settings JSON file (`appsettings.canary.json` or `appsettings.prod.json`) is available
+and its values will overwrite the ones from the default `appsettings.json` file.
+
+Here's an example of the default `appsettings.json`:
+```json
+{
+    "WindowsApplicationDriverUrl": "http://127.0.0.1:4723",
+    "PackageFamilyName": "Microsoft.Windows.DevHome.Dev_8wekyb3d8bbwe",
+    "Widget": {
+        "IdPrefix": "Microsoft.Windows.DevHome.Dev_8wekyb3d8bbwe",
+        "Provider": "CoreWidgetProvider"
+    }
+}
+```
+
+### Run tests on a different release of Dev Home (Canary, Prod, Dev)
+Edit the `Test.runsettings` file and set the appropriate value for `AppSettingsMode`. In the example below `AppSettingsMode` is set to `canary`,
```

Comment:
Do we need to be careful not to check-in changes to this file? Is there any way to not have to deal we that?

Reply:
Good question! By default, I configured this file with values that works for a user/dev, and the plan is to update the pipeline YAML template to overwrite the test parameter so it runs for `prod` instead, but that will be done separately once we figure out how to run the UI tests (effort driven by infra team). To answer your question yes, if a user updates this file because they want to run the test on a different release of Dev Home, ideally that change should not be committed because it may suddenly break new tests for a user running locally, but it's also not very critical because the pipeline would not be affected (since again those values are provided anyway) and the change can be safely reverted.